lib/ui/dashboard/presenter/current_user_presenter.dart

```diff
@@ -1,8 +1,7 @@
 import 'package:flutter_dasher/common/model/user.dart';
 import 'package:flutter_dasher/domain/data/user_data_holder.dart';
-import 'package:get_it/get_it.dart';
 import 'package:hooks_riverpod/hooks_riverpod.dart';
 
 final currentUserPresenter = StateProvider.autoDispose<User>(
-  (ref) => GetIt.instance.get<UserDataHolder>().user!,
+  (ref) => ref.watch(userDataHolderProvider).user!,
 );
```

Comment:
Should we make this one code gen as well?

Reply:
Not sure what you mean here.
I'm planning to make the `CurrentUserPresenter` a `Notifier` in the next PR.

Reply:
Then I'll be eagerly awaiting the next PR 😊

Reply:
Then I'll be eagerly awaiting the next PR 😊